### PR TITLE
Support for RadPHP bundles

### PR DESCRIPTION
**Pull request by atkrad**
Support for RadPHP bundles.

### CODE REVIEW

File: src/Composer/Installers/RadPHPInstaller.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Composer\Installers;
+
+class RadPHPInstaller extends BaseInstaller
+{
+    protected $locations = [
```
Comment (niksamokhvalov) [inline src/Composer/Installers/RadPHPInstaller.php]: PHP 5.3 not supported short syntax.

**Comment by atkrad** [comment]
@niksamokhvalov I fix it, apologies for the delay in response.

File: src/Composer/Installers/RadPHPInstaller.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Composer\Installers;
+
+class RadPHPInstaller extends BaseInstaller
+{
+    protected $locations = array(
+        'bundle' => 'bundles/{$name}/'
```
Comment (niksamokhvalov) [inline src/Composer/Installers/RadPHPInstaller.php]: Just for interest: what would happen if two authors package will be called the same?

Comment (atkrad) [inline src/Composer/Installers/RadPHPInstaller.php]: @niksamokhvalov To satisfy your curiosity, bundles are used for automatic-routing which contain actions (something like controllers). So similar routing for similar action is not meaningful! For example you cannot have two routing for Pages bundle.

All other libraries go to default directory.

**Pull request merged**